src/arrow-function/index.js

```diff
@@ -12,10 +12,14 @@ export function add(){
 
   var sum = 0;
   for (var i = 0; i < arguments.length; i++) {
-    if(arguments[i])
-      sum += arguments[i];
+    if(isNumber(arguments[i]))
+      sum += +arguments[i];
     else
       throw new Error('You can only sum numbers');
   }
   return sum;
 }
+
+function isNumber(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
```

Comment:
Thanks anyway for pointing this out! I guess you just need to remove ```parseFloat```and we're good, right?